contracts/utils/PassthroughETH.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../pcv/IPCVDeposit.sol";
+contract AavePassthroughETH {
+    address payable constant target = payable(0x5B86887e171bAE0C2C826e87E34Df8D558C079B9);
+    function deposit(uint256 _amount) external payable {
+        target.call{value: address(this).balance}("");
+        IPCVDeposit(target).deposit();
```

Comment:
Don't we need to transfer the newly minted aEth to the pcvDeposit?

Seems like we wouldn't be able to without doing an approval up front.